apache_httpd-formula/pillar.example

```diff
@@ -89,6 +89,9 @@ apache_httpd:
         Request_URI:
           ^/example$: myvariable
 
+      # boolean values are rewritten to on/off
+      RewriteEngine: true
```

Comment:
No real objections, but for someone who knows the apache httpd config file syntax, true and false look very strange.

I'd strongly recommend to use on and off in the pillar, and to forget that the formula supports true and false ;-)

(That said - if you still think this is a good idea, feel free to merge.)

Reply:
`on`/`off` in YAML translates to Python `True`/`False`.

Reply:
(and I don't want to stringify `on`/`off` in the pillar with quotes ..)